src/adapters/naf-webrtc-adapter.js

```diff
@@ -367,16 +367,22 @@ class WebrtcAdapter {
             video: false
           };
           navigator.mediaDevices.getUserMedia(mediaConstraints)
-          .then(localStream => {
-            self.storeAudioStream(self.myId, localStream);
-            self.connectSuccess(self.myId);
-          })
-          .catch(e => {
-            NAF.log.error(e);
-            console.error("Microphone is disabled due to lack of permissions");
-            self.sendAudio = false;
-            self.connectSuccess(self.myId);
-          });
+            .then(localStream => {
```

Comment:
Is the white space change a spurious diff?

Reply:
I added this line as well:

```
 localStream.getTracks().forEach(
                track => {
                  Object.keys(self.peers).forEach(peerId => { 
                  self.peers[peerId].pc.addTrack(track, localStream) 
                })
              })
```

I'll revert the whitespace diff